tauri/src/app/event.rs

```diff
@@ -119,6 +119,15 @@ pub fn unlisten(event_id: EventId) {
   })
 }
 
+/// Removes an event listener by event name.
+pub fn unlisten_by_name(event_name: impl AsRef<str>) {
```

Comment:
@lucasfernog want to inform you that I added this

Reply:
you shouldn't need that; i'll take a look at it when I review the PR.

Reply:
`remove_listener` might be a better name for this. Or at least `stop_listening`, `unbind_listener` etc.